packages/core/README.md

```diff
@@ -70,20 +70,22 @@ Det er utviklet mixins for å lette bruk av vanlige mønstre. For eksempel kan d
 
 ```scss
 .jkl-some-component {
-    @include text-sizing("small");
+    @include body-paragraph;
 }
 
 /* Resulterer i følgende css: */
+.jkl-some-component {
+    font-size: 1.25rem;
+    line-height: 2rem;
+    font-weight: 400;
+}
 @media (min-width: 0) and (max-width: 767px) {
     .jkl-some-component {
-        font-size: 1rem;
-        line-height: 1.5rem;
+        font-size: 1.125rem;
+        line-height: 1.75rem;
+        font-weight: 400;
```

Comment:
Dette likte jeg ikke. Feil dokumentasjon er verre enn ingen dokumentasjon, og dette har veldig lett for å bli feil. Hvis vi oppdater font-vekten til 300, så må man bare huske å oppdatere her, det er fort gjort at ikke skjer. Dessuten, om du vil vite hva cssen blir så kan du bare se i css fila, eller forstå scss fila.

Reply:
Vi har vektene som variabler, så kanskje oppdater til det? `font-weight: $normal-weight;`

Reply:
Still, ikke noe fan av at vi tar jobben som manuell kompiler i README filene.